br/pkg/lightning/backend/local/local.go

```diff
@@ -1498,25 +1469,30 @@ loopWrite:
 				case retryIngest:
 					region = newRegion
 					continue
+				case retryBusyIngest:
+					log.FromContext(ctx).Warn("meet tikv busy when ingest", log.ShortError(err), logutil.SSTMetas(ingestMetas),
+						logutil.Region(region.Region))
+					// ImportEngine will continue on this unfinished range
```

Comment:
https://github.com/pingcap/tidb/blob/3cb091ba185d070bcee59f84b1665fc584974631/br/pkg/lightning/backend/local/local.go#L1651-L1682

Seems there is no retry limit for ImportEngine. Are there any risks that will make lightning hang/run too long?

Reply:
I'm not sure about the old logic, PTAL @gozssky

Reply:
Yes, there is no retry limit. Maybe we can check if there is progress on each retry. If there is no progress, then set a retry limit.

Reply:
Don't want to add another retry counter and sleep interval for now. I'm thinking of providing a retry framework 😂 since the problem does not become worse than before, I prefer we do it later

Reply:
ptal @buchuitoudegou

Reply:
lgtm